src/app/utils/item.js

```diff
@@ -223,62 +248,62 @@ function LibraryItem() {
    * @param {string} itemId
    * @return {object}
    */
-  this.getDeliveryLocations = (item, single = true, bib = '', itemId = '') => {
-    const thisItem = !_isEmpty(item) ? item : this.getItem(bib, itemId);
-    // default to SASB - RMRR
-    const defaultDeliveryLocations = this.defaultDeliveryLocations();
-
-    // get location and location code
-    let deliveryLocations = defaultDeliveryLocations;
-    let locations = [];
-
-    if (thisItem) {
-      if (thisItem.deliveryLocations && thisItem.deliveryLocations.length &&
-        !(thisItem.id.substring(4) === 'i16429984')) {
-        deliveryLocations = thisItem.deliveryLocations;
-      }
-
-      deliveryLocations.forEach((location) => {
-        const locationCode = (location['@id'] && typeof location['@id'] === 'string') ?
-          location['@id'].substring(4) : '';
-        // location is set to defaultDeliveryLocations so it the following will always be false:
-        const isOffsite = this.isOffsite(location.prefLabel);
-        // retrieve delivery location
-        let deliveryLocationCode = location['@id'].substring(4);
-        let returnLocation = {};
-
-        // retrieve location data
-        if (locationCode && locationCode in LocationCodes) {
-          returnLocation = Locations[LocationCodes[locationCode].location];
-          deliveryLocationCode = LocationCodes[locationCode].delivery_location || '';
-        } else {
-          returnLocation =
-            Locations[LocationCodes[defaultDeliveryLocations[0]['@id'].substring(4)].location];
-        }
-
-        if (isOffsite && deliveryLocationCode === defaultDeliveryLocations[0]['@id'].substring(4)) {
-          returnLocation.prefLabel = defaultDeliveryLocations[0].prefLabel;
-        }
-
-        returnLocation = _extend({
-          customerCode: location.customerCode,
-          prefLabel: location.prefLabel,
-          offsite: isOffsite,
-          code: deliveryLocationCode,
-        }, returnLocation);
-
-        locations.push(returnLocation);
-      });
-    }
-
-    // Temporary for now
-    if (single) {
-      // just return the first element in the array.
-      locations = [locations[0]];
-    }
-
-    return locations;
-  };
+  // this.getDeliveryLocations = (item, single = true, bib = '', itemId = '') => {
```

Comment:
Yes, but that should be a different function since this says `delivery`. Here's a ticket for that https://github.com/NYPL-discovery/discovery-front-end/issues/513.